src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstancePropertyValue.scala

```diff
@@ -201,10 +184,22 @@ object InstancePropertyValue {
                     InstancePropertyValue.Float64List(arr.flatMap(_.asNumber).map(_.toDouble))
                 }
                 Right[DecodingFailure, InstancePropertyValue](matchingPropType)
-              case element if element.isObject =>
-                Right[DecodingFailure, InstancePropertyValue](
-                  InstancePropertyValue.ObjectList(arr)
-                )
+              case element if element.isObject && isDirectRelation(element) =>
```

Comment:
```suggestion
              case element if isDirectRelation(element) =>
```
technically should be the same